src/controllers/media.ts

```diff
@@ -18,6 +19,12 @@ export const getByOsdbHash = asyncHandler(async(req: Request, res: Response, nex
   };
 
   const osMeta: OpensubtitlesIdentifyResponse = await osAPI.identify(osQuery);
+
+  if (!osMeta.metadata) {
+    await FailedLookups.create({ osdbHash });
+    return res.status(200).json({ message: 'Metadata not found on OpenSubtitles' });
```

Comment:
I haven't looked at the client consuming this API yet, but is a 200 response code right for this? We would essentially want to tell the client to stop making this request wouldn't we, so maybe something 4xx could be better?

Reply:
Yeah I was kinda going back and forth about this in my head too. It's not really an error to not find that info, and the API has succeeded at its task of checking the OpenSubs API... I guess the definition of 404 fits `The requested resource could not be found but may be available in the future.` although it's not an error in this case

Reply:
We can think of it like a search situation - if I search for something in a search engine and get 0 results, is that an error or did I successfully find out there are no matches? I'd say the latter

Reply:
There are other status codes than 404 that could be used, that could be used to make the distinction. 425 is one example, 503 might be another: https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml